docs/content/users/quickstart.md

```diff
@@ -1,274 +1,230 @@
 # CMS Quickstarts
 
-While the generic `php` project type is [ready to go](./project.md) with any CMS or framework, DDEV offers project types for more easily working with popular platforms and content management systems:
+With the generic `php` and `python` project types DDEV is [ready to go](./project.md) with any PHP and Python based content management system (CMS) or framework. In addition there are preconfigured project types for many popular platforms and CMSes. To learn more about how to manage projects in DDEV visit the [Managing Projects](../users/usage/managing-projects.md) page.
 
-=== "Craft CMS"
+Before proceeding, make sure your installation of DDEV is up-to-date. In a new and empty project folder, using your favorite shell, run the following commands:
```

Comment:
I think _up-to-date_ would function more like an adjective, so I’d drop the hyphens in this case:

```suggestion
Before proceeding, make sure your installation of DDEV is up to date. In a new and empty project folder, using your favorite shell, run the following commands:
```

Reply:
that is totally fine with me. i was just searching on google which of the variants (with and without the dashes) was more common.